Author: Andreas Beckmann <anbe@debian.org>
Description: fix alignment requirement for sizeof(long double) == 12
 i386 has a 12-byte long double causing
 "error: requested alignment '24' is not a positive power of 2"

--- a/include/cuda/std/detail/libcxx/include/complex
+++ b/include/cuda/std/detail/libcxx/include/complex
@@ -577,7 +577,7 @@ public:
 };
 
 template<>
-class _LIBCUDACXX_TEMPLATE_VIS _LIBCUDACXX_COMPLEX_ALIGNAS(2*sizeof(long double)) complex<long double>
+class _LIBCUDACXX_TEMPLATE_VIS _LIBCUDACXX_COMPLEX_ALIGNAS(sizeof(long double) == 12 ? 2*4 : 2*sizeof(long double)) complex<long double>
 {
 #ifndef _LIBCUDACXX_HAS_COMPLEX_LONG_DOUBLE
 public:
